examples/relay-ts/src/index.ts

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {createFragmentContainer, graphql} from 'react-relay'
+
+const Hello = ({fullName}: {fullName: string}) => React.createElement('h1', null, `Hello ${fullName}`)
```

Comment:
Does the relay typescript offer provide exported fragment types like the flow stuff?

```js
import type {User_src_data} from './__generated__/User_src_data.graphql'
```

I've always been disappointed by the way the flow stuff works. Was hoping to be surprised to see a better way with TS. 😁

Reply:
Yeah it does but as far as I can tell it's just the same as in flow 😬. We might be able to point "global" types folder in Typescript if the types are generated in a `<filename>.d.js` file but I haven't looked into something like that yet. Might be worth to take a look into it after this lands while it's all fresh in my head 😄